agent/banner/banner.go

```diff
@@ -140,9 +140,9 @@ func (w *bannerResponseWriter) getFavIconLink() (string, error) {
 	}
 	var favIconLinkBuf bytes.Buffer
 	templateVals := &struct {
-		FavIconUrl string
+		FavIconURL string
```

Comment:
This attribute is already being referenced by the updated name in the [favIconLinkTemplate](https://github.com/google/inverting-proxy/blob/b78af2b2b59f4a681674c08e6b5efa025a7ba9c5/agent/banner/banner.go#L57).